src/main/java/org/mastodon/mamut/fitting/edgel/SampleEllipsoidEdgel.java

```diff
@@ -96,11 +107,15 @@ public static Ellipsoid sample(
 				}
 				final Ellipsoid ellipsoid = FitEllipsoid.yuryPetrov( coordinates );
 
-				// don't count degenerate samples
+				// skip samples that cannot be fitted to an ellipsoid
```

Comment:
The `FitEllipsoid.yuryPetrov(...)` method has several ways to fails:
1. return null
2. return an ellipsoid with radii that is NaN
3. throw IlleagalArgumentExcption
4. throw RuntimeException

This is not nice and very complex to handle. This PR doesn't introduce this problem, but very much tries to solve the consequences. You are understanding well under which conditions these errors occur, and how to handle them. Please describe it in javadoc or a comment.